docs/howto/static-files/index.txt

```diff
@@ -94,10 +94,11 @@ this by adding the following snippet to your urls.py::
 
     from django.conf import settings
     from django.conf.urls.static import static
+    from django.contrib.staticfiles.views import serve
```

Comment:
I think the line above about "you can still manually serve static files using the :func:`django.contrib.staticfiles.views.serve`" should be changed to reference django.views.static.serve as you changed in the next section -- the example looks like it's suggested how to serve files without using contrib.staticfiles.